templates/base.html

```diff
@@ -25,6 +25,10 @@
 
         </div>
         <div class="col-sm-10 ">
+            {% if search_form %}
+                {% include "includes/search.html" %}
+            {% endif %}
+
```

Comment:
In that case, all the html files that you inherit from the base.html will contain the search.
Do you think this is correct?